### ISSUE TITLE
Penumbra should have tests for {clients, conns, chans, transfer}.

### ISSUE THREAD

**Issue by boojamya** [strangelove-ventures/interchaintest#313]
```[tasklist]
### Tasks
- [ ] https://github.com/strangelove-ventures/interchaintest/issues/644
- [ ] https://github.com/strangelove-ventures/interchaintest/pull/480
```

**Comment by boojamya** [comment]
Docker image: https://github.com/strangelove-ventures/heighliner/pkgs/container/heighliner%2Fpenumbra  (030-isonoe)
Relayer image: https://github.com/penumbra-zone/relayer/pull/2

**Comment by jonathanpberger** [comment]
Blocked on `pclientd` interchain test integration #480. Pair w/ @jtieri to resolve.

**Comment by jonathanpberger** [comment]
@jtieri sez "calls are working for addresses, but getting an EOF error. Still troubleshooting." Will unblock when #480 is clear.

**Comment by jonathanpberger** [comment]
Picking these Penumbra calls back up tomorrow, hoping to finish this off finally.

**Comment by jonathanpberger** [comment]
Conner got back to @jtieri, confirmed there's an issue but they're not seeing it in their other relayer so it's a low priority. Waiting on them to let us know.

**Comment by Reecepbcups** [comment]
Completed!
